ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -54,6 +54,7 @@ conformance_test(
         ":bootstrap.canton",
         ":canton",
         ":canton.conf",
+        ":logback-debug.xml",
```

Comment:
I understand this was useful to find the issue, but I don't think it's a good idea to keep this on CI. It would mean that an error on a Canton conformance test would yield _extremely_ long and likely difficult to read logs.

Reply:
Yeah, I considered this as well - indeed the log file has 300K lines.

I will take a look at how long the log file is if I still only log canton at INFO, but set the DAML-repo classes to log at DEBUG. That ended up being key to tracking down the issue and also those logs are more familiar and easier to read for a "general daml ledger" audience.

Reply:
Ok, not logging canton at DEBUG gets the log down to about 64K while still providing insights into "reads" performed by the tests. Particularly many of the more "obscure", canton-concept lines (such as those gathering confirmation consensus) are now gone (except for the results logged at info).